exts/cesium.omniverse/cesium/omniverse/ui/main_window.py

```diff
@@ -132,8 +132,6 @@ def _setup_subscriptions(self):
         )
 
     def _on_update_frame(self, _e: carb.events.IEvent):
-        self._cesium_omniverse_interface.on_update_ui()
```

Comment:
Is this OK to delete? This calls this C++ code:

```
void Context::onUpdateUi() {
    // A lot of UI code will end up calling the session prior to us actually having a stage. The user won't see this
    // but some major segfaults will occur without this check.
    if (!UsdUtil::hasStage()) {
        return;
    }
```

Reply:
It should be ok to delete, but I need to test it some more. `onUpdateUi` was removed and `onUpdateFrame` now ticks the servers.